drivers/include/mtd.h

```diff
@@ -120,6 +141,27 @@ struct mtd_desc {
                  uint32_t addr,
                  uint32_t size);
 
+    /**
+     * @brief   Write to the Memory Technology Device (MTD) using
+     *          pagewise addressing.
+     *
+     * @p offset should not exceed the page size
```

Comment:
Is it allowed to have `@p size + @p offset > PAGE_SIZE`?, in other words, is it allowed to write into the next page?

Reply:
Yes, the low-level `read_page` function will only read to the end of the page, `mtd_read_page()` takes care of splitting it up into two reads.